lib/_debugger.js

```diff
@@ -1710,7 +1711,7 @@ Interface.prototype.trySpawn = function(cb) {
   function connectError() {
     // If it's failed to connect 10 times then print failed message
     if (connectionAttempts >= 10) {
-      console.error(' failed, please retry');
+      internalUtil.error(' failed, please retry');
```

Comment:
I think this error message could be more descriptive. If you like to do it, you can do it now.